### PR TITLE
Swapping FastImage for RMagick

### PR DESCRIPTION
**Pull request by aarongustafson**
Addresses #25

### CODE REVIEW

**Comment by aarongustafson** [comment]
Tested on one site and it worked flawlessly. Let me know how it works for you.

**Comment by nhoizey** [comment]
Sorry @aarongustafson, merging the other PR broke this one.

**Comment by aarongustafson** [comment]
Taking a quick pass through, I think this merges (and simplifies) the two. Just an FYI: I’m doing this between daddy tasks, so I haven’t had a chance to test yet, but if you wanna give it a go, I’m pretty sure I got it all stitched together properly.

**Comment by aarongustafson** [comment]
Done.

File: lib/jekyll/cloudinary.rb [hunk 1]
```diff
@@ -218,41 +217,19 @@ def render(context)
...
-            Jekyll.logger.warn(
-              "[Cloudinary]",
-              "Couldn't find this image to check its width: #{image_path}. \
-              Try to run Jekyll build a second time. If it still doesn’t work \
-              it could be that your ImageMagick install doesn’t support this format."
-            )
-          end
-        end
-
-        if preset["width_height"] == false
```
Comment (nhoizey) [inline lib/jekyll/cloudinary.rb]: You lost this feature in the process, it seems.

Comment (aarongustafson) [inline lib/jekyll/cloudinary.rb]: I ended up taking it out as it was no longer necessary. It was all working around ImageMagick’s treatment of local and remote images. FastImage doesn’t care. Are you seeing an issue?

Comment (nhoizey) [inline lib/jekyll/cloudinary.rb]: Ok, it makes sense. No issue.

**Pull request merged**